README.md

```diff
@@ -6,13 +6,13 @@ It helps you create a loosely-coupled front end application which can serve traf
 
 Shunter does not contain an API client, or any Controller logic (in the [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) sense). Instead, Shunter simply proxies requests to a back end server, then:
 
-1. If the back end wants Shunter to render the response, it returns the application state as JSON, served with a certain HTTP header. This initiates the templating process in Shunter.  
```

Comment:
Is the removal of the two spaces at the end of these lines intentional? I think that their purpose is so when the Markdown is converted to HTML they will appear as a <br> instead of as a new paragraph (and thus with a margin separating them from the previous/next element.

Reply:
It's not intentional, my editor strips spaces at the end of lines 😬 
But I don't think I can see the difference in the rendered markdown?

https://github.com/springernature/shunter/tree/rename-default-branch
vs
https://github.com/springernature/shunter